src/pages/for-companies/ForCompanies.tsx

```diff
@@ -31,17 +32,28 @@ const ForCompanies: React.FC<ForCompaniesProps> = () => {
         <li>14:15 - 18:00 - Standområdet er åpent</li>
         <li>18:00 - 20:00 - Nedrigging av lokalet</li>
       </ul>
-      <h5>Praktisk informasjon</h5>
+      <PageTitle>Praktisk informasjon</PageTitle>
+      <h5>Standområdet</h5>
       <p>
         Dere har et område på 2m * 2m (4m^2) å bruke. Lokalet kommer til å være
         fullt, så det er ikke mulig å bruke mer plass enn det. Vi skaffer et
         bord og én strømpadde for dere. Det vil også bli enkel matservering i
         form av boller.
       </p>
-      <h5>Standkart</h5>
-      <StandWrapper>
-        <StandMap src="/misc/standkart-h19.png" alt="Standkart" />
-      </StandWrapper>
+      <h5>Koble seg til uib-guest-nettet:</h5>
+      <p>
+        Vis trådløse nett på din enhet, og koble til nettet som heter
```

Comment:
Litt kronglete formulert her, og noen typos. Guiden må gå ut ifra at de har trådløst nett, vi kan ikke strekke kablet nett. Jeg ville skrevet om hele paragrafen.

```suggestion
    For å koble dere til gjestenettet på UiB bruker dere "uib-guest"
```